workflow/Snakefile

```diff
@@ -282,3 +279,24 @@ rule run_MotEvo_analysis:
         {input.DIR_pwm}) \
         1> {log.LOG_local_stdout} 2> {log.LOG_local_stderr}
         """
+
+rule combine_MotEvo_results:
+    input: 
+        DIR_MotEvo_results_pwm = expand(
+            os.path.join(
+                "{outdir}",
+                "motevo",
+                "{pwm_dir}"
+            ),
+            outdir = config["outdir"],
+            pwm_dir = gather_motifs_names(config["pwm_directory"])
+        )
+    output:
+        combined_MotEvo_results = expand(os.path.join(
+            "{outdir}",
+            "combined_MotEvo_results.tsv"
+        ),
+        outdir = config["outdir"]
+        ),
+    params: "posterior_sites"
+    script: "parse.py"
```

Comment:
Now I realised that if we call it like this then the script depends on the content of the `snakemake.`. This is bad, as we would not be able to call it outside of the pipeline ever (implying no unit tests as well).
Please change it to the `bash` call as for the other rules.

Reply:
You will have to pass arguments to the script explicitly, please take a look at the file `format-ATtRACT-motifs.py` to see how to catch CLI arguments from Python.